payloads/external/edk2/Kconfig

```diff
@@ -268,6 +268,12 @@ config EDK2_OPAL_PASSWORD
 	help
 	  Enables TCG OPAL disk password in the TianoCore Payload.
 
+config EDK2_SETUP_PASSWORD
+	bool "Enable TianoCore setup menu password"
+	default n
+	help
+	  Enables setup menu password in the TianoCore Payload.
```

Comment:
I would make it default to enabled. This is not something we want t configure I guess. So no need to specify Kconfig for it

Reply:
Done. Removed the option entirely